pop-api/examples/place-spot-order/lib.rs

```diff
@@ -1,34 +1,16 @@
 #![cfg_attr(not(feature = "std"), no_std, no_main)]
 
-use pop_api::nfts;
-
-#[derive(Debug, Copy, Clone, PartialEq, Eq, scale::Encode, scale::Decode)]
-#[cfg_attr(feature = "std", derive(scale_info::TypeInfo))]
-pub enum ContractError {
-	InvalidCollection,
-	ItemAlreadyExists,
-	NftsError(nfts::Error),
-	NotOwner,
-}
-
-impl From<nfts::Error> for ContractError {
-	fn from(value: nfts::Error) -> Self {
-		ContractError::NftsError(value)
-	}
-}
-
 #[ink::contract(env = pop_api::Environment)]
-mod pop_api_spot_order_example {
-	use super::ContractError;
+mod spot_order {
```

Comment:
I removed the error handling. This should be implemented properly in another PR if we see value in this example (e.g. as a sub0 presentation reference).